_includes/configuration/powerpaste_html_import.md

```diff
@@ -0,0 +1,31 @@
+### `powerpaste_word_import`
```

Comment:
Another file using the wrong content, this should be using the `powerpaste_html_import` content, not the `powerpaste_word_import` content.

- [ ] Fix incorrect setting content

Reply:
fixed in [`daef718` (#2038)](https://github.com/tinymce/tinymce-docs/pull/2038/commits/daef718a0fde627f3385621857d5eb387543516b)